src/commands/utility/vote-points.js

```diff
@@ -5,28 +5,48 @@ module.exports = {
   data: new SlashCommandBuilder()
     .setName('vote-points')
     .setDescription(
-      'Start a Effort Estimation Points votation for current Thread'
+      'Start an Effort Estimation Points votation for the current Thread'
     )
     .addUserOption((option) =>
       option
         .setName('user')
         .setDescription('User to search for')
         .setRequired(true)
+    )
+    .addStringOption((option) =>
+      option
+        .setName('point-type')
+        .setDescription('Select the type of points to award')
+        .setRequired(true)
+        .addChoices(
+          { name: 'Normal Points', value: 'normal' },
+          { name: 'Boosted Points', value: 'boosted' }
+        )
     ),
   async execute(interaction) {
     const user = interaction.options.getUser('user') || interaction.user;
+    const pointType = interaction.options.getString('point-type');
+
+    const selectedTagId =
+      pointType === 'boosted' ? tagIds.boostedPointTagId : tagIds.addPointTagId;
 
-    await interaction.reply(`<@&${tagIds.taskCompletedTagId}> <@${user.id}>`);
+    await interaction.reply(
+      `<@&${tagIds.taskCompletedTagId}> <@${user.id}> (Awarding: ${pointType.toUpperCase()} Points with Tag <@&${selectedTagId}>)`
+    );
 
     interaction.followUp({
       poll: {
         question: {
-          text: `How much cost this task? | ${user.id}`,
+          text: `How much does this task cost? Point type: ${pointType} | ${user.id}`,
```

Comment:
@Israel-Laguan suggested add a descriptive name before the ${user.id}, not sure if it can be added here

Reply:
what do you think about this change? @ARenzDev